site/docs/components/svg/infoIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import { cn } from '../../../utils/index.ts';
+
+const InfoIcon = ({ className }: { className?: string }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+    strokeWidth="2"
+    strokeLinecap="round"
+    strokeLinejoin="round"
+    className={cn(className)}
```

Comment:
may want to use role="img" and aria-labelledby + id on your title to help screen readers